.circleci/config.yml

```diff
@@ -838,8 +856,16 @@ jobs:
       - run:
           name: strip binary
           command: strip build/solc/solc
-      - store_artifacts: *artifacts_solc
-      - persist_to_workspace: *artifacts_executables
+      - store_artifacts:
+          path: build/solc/solc
+          destination: solc-static-linux
```

Comment:
I think you could just add `solc-static-linux` to `artifacts_solc`. IIRC artifacts that do not exist are ignored.

Reply:
The `store_artifacts` step only accepts `path` and `destination` as keys, so I cannot use a list of destinations. As far I could see, we can store the whole `build/solc` directory or we store individual files as I did.

Note the binaries are the same and the renaming was only necessary to avoid two jobs (`b_ubu_static` and `b_ubu`) to persist the same file, which would cause problems like this one: https://app.circleci.com/pipelines/github/ethereum/solidity/28500/workflows/e51bbd05-4a2b-400e-a489-c27d3b401b2f/jobs/1265263

Reply:
ah, ok then.

You should still be able to store that artifact if you keep it in `build` rather than `build-static` as I suggested elsewhere, right?

Reply:
yes, I believe so. I already changed it to `build`, so let's see :)